apps/dashboard/config/configuration_singleton.rb

```diff
@@ -250,8 +250,54 @@ def native_vnc_login_host
     ENV['OOD_NATIVE_VNC_LOGIN_HOST']
   end
 
+  # Set the global configuration file
+  def config_file
+    Pathname.new(ENV['OOD_CONFIG_FILE'] || "/etc/ood/config/config.yml")
+  end
+
+  def pinned_apps
+    @pinned_apps ||= begin
+      config[:pinned_apps].to_a.map do |token|
+        router = router_from_token(token.to_s)
```

Comment:
Ok so this PR is for the MVP mentioned https://github.com/OSC/ondemand/issues/714 presumably, 

>Admin can configure specific apps (sys/bc_osc_ansys_workbench, sys/bc_desktop/vdi),

But postponing support for a future PR to:

>Admin can configure...group of apps by type (type: usr apps, all type: sys apps, all type: dev apps), app categories/subcategories, metadata keys (field_of_science: )

?

Reply:
Yea that's right. It's sort of always my preference to start small and break up large tickets/features if we can. In this case we can.

Reply:
I went for Router because having this inside OodApp means OodApp has knowledge of the sublcass.